### PR TITLE
Redux/ujwal

### PR DESCRIPTION
**Pull request by UjwalBattar**
@artoodeeto @CyberPutty 
Cleaned up files while staying semi-true to our current format.
Added redux-logger to log stuff in browser console for us.
Added but did not use 'util' files and code for frontend auth - these will need modification, I just grabbed boilerplate from another project of mine.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import Layout from './Components/Common/Layout/index';
+
+const App = ({ store }) => (
+  <Provider store={store}>
```
Comment (artoodeeto) [inline client/src/App.jsx]: @UjwalBattar why do we have a sotre here and on the iclient/src/ndex.js file? just want to make sure.

File: client/src/index.js [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+  //   const currentTime = Date.now() / 1000;
+  //   if (decoded.exp < currentTime) {
+  //     // Logout user
+  //     store.dispatch(APIUtil.logoutUser());
+  //     // Redirect to login
+  //     window.location.href = '/login';
+  //   }
+  // }
+  const root = document.getElementById('root');
+  ReactDOM.render(<App store={store} />, root);
```
Comment (artoodeeto) [inline client/src/index.js]: @UjwalBattar this one right here.

**Comment by artoodeeto** [comment]
@UjwalBattar yow dude. sorry for holding you off but just wanted to ask.

File: client/src/App.jsx (hunk 1 above)
Comment (UjwalBattar) [reply client/src/App.jsx]: So here, we are pulling the app store and destructuring to use in our client side

File: client/src/index.js (hunk 2 above)
Comment (UjwalBattar) [reply client/src/index.js]: And here, we are actually creating and configuring the app store when the app initially loads

Comment (artoodeeto) [reply client/src/index.js]: ohhh. ok your just passing the store as a props. cool. ok man. this is good. ill merge this.

**Pull request merged**